front/src/views/Products/components/ProductsContainer.tsx

```diff
@@ -0,0 +1,187 @@
+import { useSuspenseQuery } from "@apollo/client";
+import { Badge } from "@chakra-ui/react";
+import { SelectColumnFilter } from "components/Table/Filter";
+import { useTableConfig } from "hooks/hooks";
+import { useMemo } from "react";
+import { Column, CellProps } from "react-table";
+import { PRODUCT_BASIC_FIELDS_FRAGMENT } from "../../../../../graphql/fragments";
+import { graphql } from "../../../../../graphql/graphql";
+import { ProductRow, productsRawToTableDataTransformer } from "./transformers";
+import { useAtomValue } from "jotai";
+import { selectedBaseIdAtom } from "stores/globalPreferenceStore";
+import { DateCell } from "components/Table/Cells";
+import ProductsTable from "./ProductsTable";
+import { AllProductsCell } from "./TableCells";
+
+export const PRODUCTS_QUERY = graphql(
+  `
+    query ProductsForProductsView {
+      products(paginationInput: { first: 10000 }) {
+        totalCount
+        elements {
+          ...ProductBasicFields
+          sizeRange {
+            id
+            name
+            label
+            sizes {
+              id
+              name
+              label
```

Comment:
name and label are alias field for one another, should be sufficient to go with only one